### PR TITLE
make checker for git fetch refs

### PR DESCRIPTION
**Pull request by naumvd95**
https://mirantis.jira.com/browse/FPL-2687

### CODE REVIEW

File: servers/partner-ci/9.0/manila.yaml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 ---
+- parameter:
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/manila.yaml]: should be in common-parameters

File: servers/partner-ci/9.x/vcenter.yaml [hunk 2]
```diff
@@ -7,6 +7,8 @@
       - string:
           name: MULTI_REFS
       - string:
+          name: FUEL_QA_REFS
+      - string:
           name: SPECIFIC_VAR
```
Comment (rkhozinov) [inline servers/partner-ci/9.x/vcenter.yaml]: SPECIFIC_VAR - what is that?

**Review by rkhozinov — CHANGES_REQUESTED** [review]

File: servers/partner-ci/9.0/contrail.yaml [hunk 3]
```diff
@@ -446,11 +448,20 @@
       - test-property
     builders:
       - shell: |
+          if [[ "${{FUEL_QA_REFS}}" != "none" ]]; then
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/contrail.yaml]: use macros

File: servers/partner-ci/9.x/nsxv.yaml [hunk 4]
```diff
@@ -286,11 +285,20 @@
       - test-property
     builders:
       - shell: |
+          if [[ "${{FUEL_QA_REFS}}" != "none" ]]; then
```
Comment (rkhozinov) [inline servers/partner-ci/9.x/nsxv.yaml]: get-refs

**Review by rkhozinov — CHANGES_REQUESTED** [review]

File: servers/partner-ci/common.yaml [hunk 5]
```diff
@@ -46,6 +46,10 @@
           default: '{skip_vote_failed}'
           description: "configure gerrit voting"
       - string:
+          name: FUEL_QA_REFS
+      - string:
+          name: MULTI_REFS
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml:52]: add description

File: servers/partner-ci/common.yaml [hunk 6]
```diff
@@ -46,6 +46,10 @@
           default: '{skip_vote_failed}'
           description: "configure gerrit voting"
       - string:
+          name: FUEL_QA_REFS
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml]: add description

File: servers/partner-ci/common.yaml [hunk 7]
```diff
@@ -68,7 +74,7 @@
       - string:
           name: GERRIT_REFSPEC
           default: 'refs/heads/{gerrit_branch}'
-          description: 'Refspecs for commits in fuel-qa gerrit separated with spaces'
+          description: 'Single Refspec for commits in fuel-qa gerrit'
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml]: not for fuel-qa, but for plugin repo

File: servers/partner-ci/common.yaml [hunk 8]
```diff
@@ -217,6 +223,28 @@
                     property-file: reporter.properties
                     fail-on-missing: true
                     condition: FAILED_OR_BETTER
+
+- builder:
+    name: get-refs
+    builders:
+      - shell: |
+          #!/bin/bash
+          if [[ "${FUEL_QA_REFS}" != "none" ]]; then
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml]: [ -z ${FUEL_QA_REFS} ]

File: servers/partner-ci/common.yaml (hunk 7 above)
Comment (naumvd95) [reply servers/partner-ci/common.yaml]: done

**Review by rkhozinov — APPROVED** [review]

**Comment by rkhozinov** [comment]
closes #21

**Pull request merged**